Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.ts

```diff
@@ -13,6 +14,8 @@ export class AppComponent {
   weather = 50;
   speed = 0;
   battery = 0;
+  Sinput = new FormControl(0, [Validators.required, Validators.min(0), Validators.max(90)]);
+  Binput = new FormControl(0, [Validators.required, Validators.min(0), Validators.max(100)]);
```

Comment:
These variable names are a bit vague. You can change them so that they are a bit more descriptive (i.e. speedInput, etc.)